src/main/java/mate.academy/service/ProductService.java

```diff
@@ -1,8 +1,10 @@
 package mate.academy.service;
 
 import java.util.List;
+import mate.academy.lib.Component;
 import mate.academy.model.Product;
 
+@Component
```

Comment:
Consider removing the `@Component` annotation from the interface and applying it to the implementation class instead. Annotations like `@Component` are generally used on concrete classes to indicate that they are candidates for dependency injection.